src/command/cmd_ac.c

```diff
@@ -920,6 +923,10 @@ cmd_ac_init(void)
     autocomplete_add(pgp_sendfile_ac, "on");
     autocomplete_add(pgp_sendfile_ac, "off");
 
+    pgp_autoimport_ac = autocomplete_new();
```

Comment:
For autocompletes that only support on/off you can actually use `_cmd_ac_complete_params()`.